driver-core/src/main/com/mongodb/internal/operation/CommandBatchCursor.java

```diff
@@ -263,9 +246,6 @@ private void getMore() {
                 throw translateCommandException(e, serverCursor);
             }
             resourceManager.setServerCursor(nextServerCursor);
-            if (limitReached() || !resourceManager.operable()) {
```

Comment:
Does it mean that if the resource manager is in the `CLOSE_PENDING` state after a call to `#close`, then the resources will never be released?  If so, I'm not sure this is a desirable behavior change.

Reply:
The `!resourceManager.operable()` check here wasn't needed as @stIncMale proved in mongodb/mongo-java-driver#1230 (JAVA-5124). So there wasn't a bug just a bad test - which was also updated as part of this change.

TLDR; endOperation handles closing / releasing of resources.

Reply:
I see.  Thanks